### PR TITLE
feat: connect NodeSetup fields

### PR DESCRIPTION
**Pull request by ryanio**
This PR connects the NodeSetup client option fields, please use in conjunction with [mist-shell#51](https://github.com/ethereum/mist-shell/pull/51)

screen shot 2019-02-19 at 12 22 34 am

### CODE REVIEW

File: src/containers/Network/NodeSetup/ClientConfigForm.jsx [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    // status: {},
+    options: {
+      networks: ['main', 'ropsten', 'rinkeby'],
+      ipcModes: ['IPC', 'WebSockets'],
+      syncModes: ['light', 'fast', 'full']
+    }
+  }
+
+  componentDidMount() {
+    this.getStatus()
```
Comment (marcgarreau) [inline src/containers/Network/NodeSetup/ClientConfigForm.jsx]: Breaking here - function commented out.

File: src/containers/Network/NodeSetup/ClientConfigForm.jsx [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    const newConfig = { ...config, host: event.target.value }
+    this.setState({ config: newConfig })
+    geth.setConfig(newConfig)
+  }
+
+  handleChangePort = event => {
+    const { config } = this.state
+    const newConfig = { ...config, port: Number(event.target.value) }
+    this.setState({ config: newConfig })
+    geth.setConfig(newConfig)
```
Comment (marcgarreau) [inline src/containers/Network/NodeSetup/ClientConfigForm.jsx]: promoting one Source of Truth™, `geth.setConfig` could be called only when start/stop btn clicked.

File: src/containers/Network/NodeSetup/ClientConfigForm.jsx (hunk 1 above)
Comment (ryanio) [reply src/containers/Network/NodeSetup/ClientConfigForm.jsx]: fixed, removed unneeded code

File: src/containers/Network/NodeSetup/ClientConfigForm.jsx (hunk 2 above)
Comment (ryanio) [reply src/containers/Network/NodeSetup/ClientConfigForm.jsx]: ty

**Pull request merged**